### PR TITLE
Feature/e UI 8525

### PR DESCRIPTION
**Pull request by MateiHMCTS**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EUI-8525

### Change description ###
Continue to raise a query and validation

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

**Review by johnbenjamin-hmcts — APPROVED** [review]

File: src/cases/containers/query-management-container/query-management-container.component.ts [hunk 1]
```diff
@@ -123,13 +176,31 @@ export class QueryManagementContainerComponent implements OnInit {
...
 
-  public navigateToErrorElement(elementId: string): void {
-    if (elementId) {
-      const htmlElement = document.getElementById(elementId);
-      if (htmlElement) {
-        htmlElement.scrollIntoView({ behavior: 'smooth', block: 'center' });
-        htmlElement.focus();
-      }
+  private getQueryCreateContext(queryItemId: string): QueryItemType {
+    if (!queryItemId) {
```
Comment (DavidJayakumar) [inline src/cases/containers/query-management-container/query-management-container.component.ts:180]: Is this conditon required? Will this not be handled within switch default?

**Review by DavidJayakumar — COMMENTED** [review]
@MateiHMCTS Added a comment

File: src/cases/containers/query-management-container/query-management-container.component.ts (hunk 1 above)
Comment (johnbenjamin-hmcts) [reply src/cases/containers/query-management-container/query-management-container.component.ts:180]: @DavidJayakumar it is part of the code I wrote for https://tools.hmcts.net/jira/browse/EUI-8523 and yes you are correct, the default is hit when the `queryItemId` is undefined.

I will handle this as part of my current ticket https://tools.hmcts.net/jira/browse/EUI-8524.

**Review by DavidJayakumar — APPROVED** [review]
Looks good

**Pull request merged**